workspaces/src/network/variants.rs

```diff
@@ -15,6 +15,7 @@ pub trait NetworkInfo {
     fn info(&self) -> &Info;
 }
 
+#[deprecated = "only the registrar can create top level accounts as of Protocol >=64"]
```

Comment:
Top-level accounts creation is not deprecated, so I am not completely convinced that we should mark this trait as deprecated

Reply:
I added that as an attempt to discourage usage, maybe we use `warn`?